docker/test/integration/MiNiFi_integration_test_driver.py

```diff
@@ -60,6 +60,12 @@ def start_kafka_broker(self):
         self.cluster.deploy('kafka-broker')
         assert self.wait_for_container_startup_to_finish('kafka-broker')
 
+    def start_splunk(self):
+        self.cluster.acquire_container('splunk', 'splunk')
+        self.cluster.deploy('splunk')
+        assert self.wait_for_container_startup_to_finish('splunk')
+        assert self.cluster.enable_hec_indexer('splunk', 'splunk_hec_token')
```

Comment:
It could be avoided if we would keep the source file and repeatedly rerun the flow until it succeeds, but maybe this solution makes the test case more stable and reliable. I'm okay with keeping it like this.